### PR TITLE
add panel 4G - tp53/telomerase FP

### PR DESCRIPTION
**Pull request by jharenza**
#### What scientific question is your analysis addressing?
Replacing https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1343 because of merge conflict issues

#### What was your approach?
Create forest plot for tp53 and telomerase scores with additional covariates of LGG group, HGG group, and extent of tumor resection

#### What GitHub issue does your pull request address?
NA

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: figures/scripts/fig4-tp53-telomerase-panels.R [hunk 1]
```diff
@@ -416,28 +422,64 @@ dev.off()
...
   select(n, nevent)
 
 # Convert survival model result to data frame, and exponentiate estimates/CIs to get HRs
 survival_df <- broom::tidy(survival_result) %>%
+  # add references
+  add_row(term = resect_ref, estimate = 0) %>%
+  add_row(term = lgg_ref, estimate = 0) %>%
+  add_row(term = hgg_ref, estimate = 0) %>%
+  #remove unknown resection from plot
+  filter(term != "extent_of_tumor_resectionUnavailable") %>%
```
Comment (sjspielman) [inline figures/scripts/fig4-tp53-telomerase-panels.R:462]: Why is this being removed from the plot? It's definitely part of the model [here](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/survival-analysis/survival-analysis_tp53_telomerase.nb.html#additive-model-tp53_scoretelomerase_scoreextent_of_tumor_resectionlgg_grouphgg_group).

Comment (sjspielman) [reply figures/scripts/fig4-tp53-telomerase-panels.R:462]: Noting we discussed that this term should actually not be in the model, so this PR has been filed to take it out of the model, which will facilitate this plot soon - https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1356

Comment (jharenza) [reply figures/scripts/fig4-tp53-telomerase-panels.R:462]: @sjspielman the FP has been rerun with #1356 in

**Review by sjspielman — APPROVED** [review]
👍

**Comment by sjspielman** [comment]
These changes do not require CI, so we can merge before checks are complete.

**Pull request merged**